README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# **Jessica Lee** (*preferred*: Jess)
+
+![This is me!](/Users/JessicaLee/Desktop/Jess.jpg)
```

Comment:
HI Nadiah (And Jessica). We actually **DO NOT** recommend you upload images through github directly. That can create many problems in more complicated, collaborative projects so we try not to encourage the habit here. To include your image in your project repo, go to RStudio server where you wrote your README for this project and click the upload button in the bottom right hand corner by your file manager. There, you can upload a file to your project, and then commit and push up the image through RStudio directly instead of through GitHub. That will also allow you to use a relative filepath instead of one that is defined on your computer. My users folder is "Neeraj Sharma" so trying to find an image through a "JessicaLee" directory won't work!